robosistance/src/endpoints/db.rs

```diff
@@ -0,0 +1,14 @@
+use crate::{models::robot_model::RobotData, db::mongodb_robot::MongoRepo};
+use mongodb::{bson::oid::ObjectId, results::InsertOneResult};
+use rocket::{http::Status, serde::json::Json, State};
+use rocket::get;
+
+#[get("/get_robot_data")]
+pub fn get_robot_data(db: &State<MongoRepo>) -> Result<Json<Vec<RobotData>>, Status> {
+    println!("COMES HERE");   
```

Comment:
Remove this print. You could add an appropriate log if you wish but I don't think it's needed. This seems to be for testing only.